Tasks/ServiceFabricDeployV1/utilities.ps1

```diff
@@ -152,7 +152,6 @@ function Get-VstsUpgradeParameters
 
     $parameterNames = @(
         "UpgradeReplicaSetCheckTimeoutSec",
-        "ReplicaQuorumTimeoutSec",
```

Comment:
I am fine with removing the input, but this can be a breaking change for existing release definitions where only ReplicaQuorumTimeoutSec is set. Ideally the same value should be set for UpgradeReplicaSetCheckTimeoutSec. Agree?
We can can remove `UpgradeReplicaSetCheckTimeoutSec` from `$parametersName` array and do something like:
```
$upgradeReplicaSetCheckTimeoutSecValue = Get-VstsInput -Name "UpgradeReplicaSetCheckTimeoutSec"
if(!$upgradeReplicaSetCheckTimeoutSecValue)
{
    $upgradeReplicaSetCheckTimeoutSecValue = Get-VstsInput -Name "ReplicaQuorumTimeoutSec"
}

if($upgradeReplicaSetCheckTimeoutSecValue)
{
    $parameters["UpgradeReplicaSetCheckTimeoutSec"] = $upgradeReplicaSetCheckTimeoutSecValue
}
```

Reply:
Makes sense, I agree. I'll make that change within the next few days.